provider/core.rb

```diff
@@ -412,7 +416,7 @@ def format_output_file(path)
 
     def run_formatter(command)
       output = %x(#{command} 2>&1)
-      Google::LOGGER.error output if $CHILD_STATUS && $CHILD_STATUS.exitstatus != 0
+      Google::LOGGER.error output unless $CHILD_STATUS&.exitstatus&.zero?
```

Comment:
This was suppressing a lot of black errors + is more Ruby (to the best of my knowledge)